python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -43,8 +54,8 @@ def __init__(self, driver=None, host=None, port=None, database=None):
                     stored in sims_data/OpSimData/
         """
         if database is None:
```

Comment:
The unittests are there to tell us if we broke something.  I say remove it and see what happens.  If the concern is that you don't have a 100% cloned version of the stack installed to test on, then I can do it after you make the other changes on this branch.